src/oaklib/implementations/funowl/funowl_implementation.py

```diff
@@ -97,7 +97,7 @@ def label(self, curie: CURIE, lang: Optional[LANGUAGE_TAG] = None) -> str:
             else:
                 raise ValueError(f"Label must be literal, not {label}")
 
-    def entities(self, filter_obsoletes=True, owl_type=None) -> Iterable[CURIE]:
+    def entities(self, filter_obsoletes=True, owl_type=None) -> Iterable[Union[URI, CURIE]]:
```

Comment:
## The update
This is a minor update. I just happened to notice that while I was using this for `SqlImplementation`, I was getting back URIs sometimes. I assume this should always be a possible thing for the other implementations as well.

Reply:
Thanks @joeflack4 !